sre-recipes/recipes/recipe_1/currencies_recipe.py

```diff
@@ -0,0 +1,138 @@
+        logging.info('Deleting pod: %s', delete_pod_command)
+        CurrenciesRecipe._run_command(delete_pod_command)
+
+    def break_service(self):
+        """
+        Rolls back the working version of the given service and deploys the
+        broken version of the given service
+        """
+        print("Deploying broken service...")
+        self._deploy_state(True)
+        logging.info('Deployed broken service')
+
+    def restore_service(self):
+        """
+        Rolls back the broken version of the given service and deploys the
+        working version of the given service
+        """
+        print("Deploying working service...")
+        self._deploy_state(False)
+        logging.info('Deployed working service')
+
+    @staticmethod
+    def _get_user_input():
+        """Prompts users for an answer"""
+        answer = input('Your answer: ')
+        return answer
+
+    def _service_multiple_choice(self):
+        """
+        Displays a multiple choice quiz to the user about which service
+        broke and prompts the user for an answer
+        """
+        print('Which service broke?')
+        print('\t[a] ad service')
+        print('\t[b] cart service')
+        print('\t[c] checkout service')
+        print('\t[d] currency service')
+        print('\t[e] email service')
+        print('\t[f] frontend service')
+        print('\t[g] payment service')
+        print('\t[h] product catalog service')
+        print('\t[i] recommendation service')
+        print('\t[j] redis')
+        print('\t[k] shipping service')
+        answer = self._get_user_input()
+        return answer
+
+    def _cause_multiple_choice(self):
+        """
+        Displays a multiple choice quiz to the user about the cause of
+        the breakage and prompts the user for an answer
+        """
+        print('What caused the breakage?')
+        print('\t[a] failed connections to other services')
+        print('\t[b] high memory usage')
+        print('\t[c] high latency')
+        print('\t[d] dropped requests')
+        answer = self._get_user_input()
+        return answer
+
+    def _verify_broken_service(self):
+        """Verifies the user found which service broke"""
+        answer = self._service_multiple_choice()
+        while answer not in ('f', 'F'):
+            print('Incorrect. Please try again.')
+            answer = self._get_user_input()
+        print('Correct! The frontend service is broken.')
+
+    def _verify_broken_cause(self):
+        """Verifies the user found the root cause of the breakage"""
+        answer = self._cause_multiple_choice()
+        while answer not in ('c', 'C'):
+            print('Incorrect. Please try again.')
+            answer = self._get_user_input()
+        print('Correct! High latency caused the breakage.')
+
+    def verify(self):
+        """Verifies the user found the root cause of the broken service"""
+        print('This is a multiple choice quiz to verify that you\'ve')
```

Comment:
nit: using double quotes here would avoid the `\`

Reply:
fixed!